Backend/Models/Database/User.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Backend.Models.Database
+{
+    public class User
+    {
+        [Key]
+        [Required]
+        public int Id { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string Name { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string Surname { get; set; }
+
+        [Required]
+        public DateTime RegistrationDate { get; set; }
+
+        [Required]
+        public WorkerStateId WorkerStateId { get; set; }
+
+        [Required]
+        public WorkerState WorkerState { get; set; }
```

Comment:
Maybe it would be more correct to name this `EmployeeState`?

Reply:
Fixed